contracts/script/SetUpEigenDA.s.sol

```diff
@@ -10,6 +10,8 @@ import {StakeRegistry} from "eigenlayer-middleware/StakeRegistry.sol";
 import {IIndexRegistry} from "eigenlayer-middleware/interfaces/IIndexRegistry.sol";
```

Comment:
is this script only used for testing?
cc @0x0aa0

Reply:
yes!